LoRaEngine/LoraKeysManagerFacade/DeviceGetter.cs

```diff
@@ -119,18 +120,17 @@ public async Task<List<IoTHubDeviceInfo>> GetDeviceList(string devEUI, string ga
                     throw new DeviceNonceUsedException();
                 }
             }
-            else if (devAddr != null)
+            else if (devAddr is { } da)
```

Comment:
This makes it worse for me to read. Now I need to understand that da is the devAddr, that was passed in. If we keep the simple != null and use devAddr, things are easier to read.

Reply:
How do I tell the compiler in that case that `devAddr` is indeed not null? It has type `DevAddr?`

Reply:
I agree with Patrick. `da` is not readable and somewhat arbitrary. I would recommend the convention of adding the `some` prefix (borrowing from optional types, which is what nullable value types are) so for `devAddr`, that would become `someDevAddr`. Another (less preferrable) option would be to add the `Value` suffix, as in `devAddrValue`. The `some` prefix is preferred because for `value` (like in the case of property setters where you don't have a choice to name the parameter), `valueValue` looks odd compared to `someValue`.

Reply:
I addressed the bad variable name with b14b97814bc6c62552dea274ac3414defca92fcc, but this doesn't solve the original comment. Should I still change something?

Reply:
My 2 cents…it solves the original problem. It's harder to see the correlation between `devAddr` and `da` than between `devAddr` and `someDevAddr`.

Reply:
Works for me. Atif and me had a discussion about the general use of this pattern and one thing that convinces me is that it makes the pattern the same for both, nullable value and ref types. This way it is consistent and you can avoid the use of .Value.